packages/velo-external-db-core/lib/router.js

```diff
@@ -26,27 +28,36 @@ const initServices = (_schemaAwareDataService, _schemaService, _operationService
     filterTransformer = _filterTransformer
     aggregationTransformer = _aggregationTransformer
     roleAuthorizationService = _roleAuthorizationService
-    hooks = _hooks || {}
+    dataHooks = _hooks?.dataHooks || {}
+    schemaHooks = _hooks?.schemaHooks || {}
```

Comment:
This exact logic is done in `app.js`, seems like an unnecessary duplication.
I would remove the change you made in `app.js` and keep this one